### PR TITLE
Project Chatbot Jennifer

### PR DESCRIPTION
**Pull request by jenster5**
https://chat-bot-technigo.netlify.app/

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -8,27 +8,30 @@
...
       rel="stylesheet" />
-    <title>Chat Bot</title>
+    <title>Vacation Bot</title>
   </head>
 
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
+    <h1>Sunny Destinations</h1>
+    <h2>Welcome to your dream vacation! </h2>
```
Comment (MaryZaia) [inline code/index.html:16]: I love your idea for the chatbot!

File: code/script.js [hunk 2]
```diff
@@ -29,23 +30,91 @@ const showMessage = (message, sender) => {
...
+event.preventDefault ()
+const nameInput = document.getElementById("name-input").value
+console.log(nameInput)
+showMessage(nameInput, "user")
+setTimeout(()=> showMessage(`Hello ${nameInput} nice to meet you. Please choose from the vacation options below.` , "bot"),1000)
+nameInput.value = ""
+setTimeout(()=> buttonsFunction(), 1000)
+});
+  
+// clicking on buttons
```
Comment (MaryZaia) [inline code/script.js:76]: Good that you have comments so one can follow along the code.

File: code/script.js [hunk 3]
```diff
@@ -29,23 +30,91 @@ const showMessage = (message, sender) => {
...
+console.log("iclicked on the Maldives button")
+showMessage("Maldives", "user")
+setTimeout(()=> showMessage("Good decision check your inbox and soon you will be on your way to paradise. ","bot"),1000)
+pickVacation()
+inputWrapper.innerHTML = ""
+
+
+} 
+else if(selectedValue.id==="Swedish"){
+console.log("iclicked on the Swedish button")
```
Comment (MaryZaia) [inline code/script.js:113]: could the console.log maybe be deleted once you know the function works?

File: README.md [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
 ## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+We were given the base code in HTML and CSS and needed to mostly work in Javascript to get the chat bot going. 
+First I linked all the elements of the HTML with the JS in the DOM section and then tried to get a  very simple chat bot going. I planned more complicated elements but could not achieve them with my level of knowledge and time given. I would have liked the boring vacation button to loop the chat back to the beginning somehow and also to have a dropdown menu of some kind. 
 
 ## View it live
-
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://app.netlify.com/sites/chat-bot-technigo/overview
```
Comment (MaryZaia) [inline README.md]: The link to the deployed site is not working.

**Review by MaryZaia — COMMENTED** [review]
Great work Jennifer! 
Please see if you can update the link to the deployed site so I can interact with you bot and book a dream vacation :)

File: README.md (hunk 4 above)
Comment (jenster5) [reply README.md]: Great thanks! I fixed this

**Pull request closed without merging**